python/pytest.ini

```diff
@@ -1,8 +1,10 @@
 [pytest]
 markers =
-  unit: marks unit tests
-  quality: marks quality tests
-  stress: marks stress tests
-  mg: marks a test as multi-GPU
-  memleak: marks a test as a memory leak test
+  unit: Quickest tests focused on accuracy and correctness
```

Comment:
will this alert us if we use an unlisted marker?

Reply:
Pytest does by default. It's probably been ignored up to this point since pytest generates so much output in our CI. You can also use `--strict-markers` to error out on markers that are not listed here instead of just generating a warning.

Reply:
we might want to create a github issue to do a review of the markers and then enable `strict-markers` to clean up things

Reply:
Yeah, could be a nice option. This (unusual marks) has not been a major issue traditionally, so I don't think it's an urgent concern.